app/src/main/res/xml/book_marks_widget_info.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<appwidget-provider xmlns:android="http://schemas.android.com/apk/res/android"
+    android:initialKeyguardLayout="@layout/book_marks_widget"
+    android:initialLayout="@layout/book_marks_widget"
+    android:minHeight="110dp"
+    android:minWidth="180dp"
+    android:previewImage="@drawable/bookmarks_appwidget_preview"
+    android:resizeMode="horizontal|vertical"
+    android:updatePeriodMillis="5000"
```

Comment:
This is too frequent, please put an appropriate period. Anything below 30 mins is going to get ignored anyway.